arbitrum-docs/build-decentralized-apps/how-to-use-timeboost.mdx

```diff
@@ -0,0 +1,425 @@
+---
+title: 'How to use Timeboost'
+description: Learn how to use timeboost
+author: jose-franco
+content_type: how-to
+---
+
+Timeboost is a new transaction ordering policy for Arbitrum chains where any participant can bid for the right to access an express lane on the sequencer for faster transaction inclusion.
+
+In this how-to, you'll learn how to bid for the right to use the express lane, how to submit transactions to it, and how to transfer that right to someone else. To learn more about Timeboost, refer to the gentle introduction.
+
+This how-to assumes that you're familiar with:
+
+- [How Timeboost works](#)
+- [viem](https://viem.sh/), since the snippets of code present in the how-to use this library
+
+## How to submit bids for the right to be the express lane controller
+
+To use the express lane for faster transaction inclusion, we must win an auction for the right to be the express lane controller for a specific round.
+
+::::info
+
+Remember that, by default, each round lasts 60 seconds, and the auction for a specific round closes 15 seconds before the round starts. These default values can be configured in a chain with the `roundDurationSeconds` and `auctionClosingSeconds` parameters respectively.
+
+::::
+
+Auctions are held in an auction contract, and bids are submitted to an autonomous auctioneer, that also communicates with the contract. Let's take a look at the process of submitting bids and finding out the winner of an auction.
+
+### Step 0: pre-requirements
+
+To run the following steps, we'll need this information:
+
+- Address of the auction contract
+- Endpoint of the autonomous auctioneer
+
+### Step 1: deposit funds into the auction contract
```

Comment:
I used this order because if you have enough balance in the contract, you don't need to get the bidding token. So since getting the bidding token and making a deposited are part of the same operation, it makes sense to put them together.